_infra/helm/locust/locustfiles/locustfile.py

```diff
@@ -146,7 +146,7 @@ def post_collection_exercise(data, url, auth):
 
 # Collection exercise event loading
 def load_collection_exercise_events(auth):
-    config = json.load(open("/mnt/locust/collection-exercise-event-config.json"))
+    config = json.load(open(".//collection-exercise-event-config.json"))
```

Comment:
The existing /mnt/ file location is specific to the containerised helm deployment. The change here is to allow the file to be references from a local non-containerised workspace.